projects/packages/jetpack-mu-wpcom/src/features/replace-site-visibility/launch-site/index.tsx

```diff
@@ -54,14 +55,17 @@ const LaunchSite = ( {
 							0
 					  ) }
 			</p>
-			<a
-				role="button"
-				className="button-secondary"
+			<button
```

Comment:
using a <button> element directly here instead of the react component because the appearance differs slightly and we don't need the extras provided by the component version in this case.